### PR TITLE
fix the label for small download counts

### PR DESCRIPTION
**Pull request by Fil**
| before | after |
|---|---|
| before | after |

### CODE REVIEW

File: src/@[scope]/[name].md.js [hunk 1]
```diff
@@ -179,18 +179,23 @@ function reviver(key, value) {
       x: {axis: "top", grid: true},
       y: {domain: d3.sort(d3.sort(versions.filter((d) => d.downloads > 0), (d) => -d.downloads).slice(0, 10).map((d) => d.version), (a, b) => semverCompare(b, a))},
       marks: [
-        Plot.barX(versions.filter((d) => d.downloads > 0), {
+        Plot.barX(versions, {
```
Comment (mbostock) [inline src/@[scope]/[name].md.js:182]: I wanted to omit versions without any downloads from the y-axis.

**Review by mbostock — CHANGES_REQUESTED** [review]

File: src/@[scope]/[name].md.js (hunk 1 above)
Comment (Fil) [reply src/@[scope]/[name].md.js:182]: the explicit domain filters them out

**Review by mbostock — APPROVED** [review]

**Comment by mbostock** [comment]
Should the logic be based on the width of the chart and/or the length of the label, too? Seems a little conservative in this example — most of these would fit in the bar.

Screenshot 2024-10-18 at 4 10 03 PM

We could also bump the contrast of the outside-the-bar labels.

**Comment by Fil** [comment]
My intention was to consider the smaller chart width, but it can be based on width for sure.

For contrast, I wanted to change the color because when it's the same white as the axis it creates visual confusion… but maybe we could use the accent color (focus) instead of gray?

**Comment by mbostock** [comment]
For contrast, we just need to use `-muted` instead of `-faint`. The latter is only intended for borders and rules; text should always be `-muted`. Using `-focus` would draw too much attention to the low values and should only be used to focus attention.

**Pull request merged**